src/styles/index.scss

```diff
@@ -1,3 +1,52 @@
+@import '/src/styles/variables';
+
 body {
+  @include center-all;
+
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc((7 * $field-side) + (6 * $calendar-gap));
+  gap: $calendar-gap;
+
+  @each $start-day, $count in $days {
+    &--start-day-#{$start-day} :first-child {
+      margin-left: calc(($field-side + $calendar-gap) * $count);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
```

Comment:
In these lines, you're hiding the extra days of the calendar based on the month length. The logic seems to be correct, but the selector might not work as expected. The ':nth-child' pseudo-class should be placed right after the parent selector without any space. Please change '&--month-length-#{$i} :nth-child(n + #{$i + 1})' to '&--month-length-#{$i}:nth-child(n + #{$i + 1})'.